query-engine/core/src/interpreter/expression.rs

```diff
@@ -11,7 +11,7 @@ pub enum Expression {
     },
 
     Query {
-        query: Query,
+        query: Box<Query>,
```

Comment:
What's the value of the `Box` here? To reduce the amount of copying that needs to happen?

Reply:
It's probably a large struct, that will make compilation slower and binary size bigger. Now we store just a pointer here, and keep the query in the heap out from the stack.